pkg/payload/payload.go

```diff
@@ -103,11 +104,47 @@ type Update struct {
 }
 
 func LoadUpdate(dir, releaseImage string) (*Update, error) {
+	return LoadUpdateWithFilters(dir, releaseImage, nil, nil)
+}
+
+type regexFilter []*regexp.Regexp
+
+func newRegexFilter(expr []string) (regexFilter, error) {
+	result := []*regexp.Regexp{}
+	for _, e := range expr {
+		re, err := regexp.Compile(e)
+		if err != nil {
+			return nil, err
+		}
+		result = append(result, re)
+	}
+	return regexFilter(result), nil
+}
+
+func (f regexFilter) matches(s string) bool {
+	for _, re := range f {
+		if re.MatchString(s) {
+			return true
+		}
+	}
+	return false
+}
+
+func LoadUpdateWithFilters(dir, releaseImage string, excludes, includes []string) (*Update, error) {
```

Comment:
the name is LoadUpdateWithFilters, but it doen't take any filters though...

Reply:
It does (`excludes`, `includes`)